validator/validator-main.protoascii

```diff
@@ -2715,6 +2715,16 @@ attr_lists: {
   attrs: { name: "itemref" }
   attrs: { name: "itemscope" }
   attrs: { name: "itemtype" }
+  # Also for schema.org, RDFa syntax
+  attrs: { name: "about" }
+  attrs: { name: "content" }
```

Comment:
All of these are fine, except `content`. We have some specific rules related to the `content` attribute for `<meta>` tags that we need to make sure we still are enforcing, for example `<meta name=viewport content=...>`.

I think the non-global attribute rules will override the global ones, but if we are adding this, it would be good to add a test which demonstrates this. Can you add a feature test showing that this fails validation:

`<meta name=viewport content=invalid>`